src/App.svelte

```diff
@@ -1,85 +1,146 @@
-<script context="module">
-	import Swipe from '@alaskaairux/ods-toast/dist/swipe.js';
-	window.Swipe = Swipe;
-</script>
-
 <script>
-	import Toaster from '@alaskaairux/ods-toast/dist/toaster';
-	import '@alaskaairux/ods-toast/dist/toaster.css';
-
-	const toaster = new Toaster();
+	let formData = {};
 
-	let type = 'primary';
-	$: message = type === 'primary' ? 'message 1' : 'message 2';
+	const flierOptions = [
+		{
+			'id': 'flierYes',
+			'label': 'Yes',
+			'value': true
+		},
+		{
+			'id': 'flierNo',
+			'label': 'No',
+			'value': false
+		}
+	];
 
-	let options = [
+	const destinationOptions = [
 		{
-			id: 'cbx1',
-			value: 'yes',
-			label: 'Yes',
-			checked: false
+			'label': 'Lower 48',
+			'value': 'lower48'
 		},
 		{
-			id: 'cbx2',
-			value: 'no',
-			label: 'No',
-			checked: false
+			'label': 'Alaska',
+			'value': 'alaska'
+		},
+		{
+			'label': 'Hawaii',
+			'value': 'alaska'
+		},
+		{
+			'label': 'Canada',
+			'value': 'canada'
+		},
+		{
+			'label': 'Mexico',
+			'value': 'mexico'
 		}
 	];
 
-	$: selectedOptions = options.filter(option => option.checked).map(option => option.value);
+	function calcDestinations(e) {
```

Comment:
I think there's a more Svelte-y way to write this stuff. Mind if I take a stab at it?